docs/gettingstarted/structuring.md

```diff
@@ -25,6 +25,84 @@ What is `@Mod`?
 
 This is an annotation indicating to the Forge Mod Loader that the class is a Mod entry point. It contains various metadata about the mod. It also designates the class that will receive `@EventHandler` events. More information can be found at... (Coming Soon)
 
+All components of @Mod() are written inside the parantheses and chained together. There's typically only one @Mod in your mod as a whole, sitting on top of your entry class.
+
+Components:
+
+`modid` [Required]
+This identifies your mod uniquely to Forge and should probably be the first thing you enter. It's only used internally, but can be seen by users who dig around. Other mods also require this ID if they want to find each other (APIs for example).
+If you want to know more about annotations in general you can check out [Oracle's docs](https://docs.oracle.com/javase/tutorial/java/annotations/index.html) on it.
+
+Example: `@Mod(modid="examplemod")`
```

Comment:
Personally, as someone who is using wikis a lot; having to read through basics of the programming language I'm developing in is becoming a nuisance. I'm here to learn about your API, not the language I'm using. I know complete beginners probably appreciate it, but for all the people that are already familiar with the language it's not adding anything but more to read. Less is more in this case, in my opinion.